Back-End/controllers/registration_controllers/register_controllers.js

```diff
@@ -1,5 +1,44 @@
-const db = require('../../db');
+const connection = require('../../db');
+require("dotenv").config();
+const bcrypt = require('bcrypt');
+
+const  register = async (req,res)=>{
+    let name=req.body.name;
+    let adress = req.body.adress;
+    let email =req.body.email;
+    let password =req.body.password;  
+    let phone = req.body.phone;
+    let role_id =0
+
+    //setting the role for the registered account
+    if(req.params.role ==="instructor"){
+        role_id = 2
+     }else if(req.params.role ==="student"){
+        role_id = 3
+     };
```

Comment:
explain this.

Reply:
if the user used the "instructor" register, his role_id will be 2 , and the student  role_id will be3